### ISSUE TITLE
batch sizes don't match paper

### ISSUE THREAD

**Issue by jcpeterson** [github-pengge/PyTorch-progressive_growing_of_gans#4]
Why are the default batch sizes used? The original paper uses 16 for sizes 4x4 to 128x128, which should be faster (overall) than what is currently used.

**Comment by github-pengge** [comment]
I did not use the same batch size as the paper used cause I ran the code on 1080 GPU with only 8GB memory, not P100, which has 16GB memory. Besides, the dataset is also different, I used CelebA dataset(with cropped and aligned), and now I'm switching to CelebA-HQ.

**Comment by jcpeterson** [comment]
I see. I also noticed the learning rate seems a bit fast to use for both the G and D. Any reason why?

**Comment by github-pengge** [comment]
I found that learning rate was not fixed in official code, and I'm changing the scheduler of learning rate now.
